Hmm, some errors when building pipewire:
```
In file included from ../spa/plugins/libcamera/libcamera-source.cpp:188:
../spa/plugins/libcamera/libcamera-utils.cpp:719:9: error: 'Transform' does not name a type
  719 |         Transform libcamera_transform;
      |         ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:722:11: error: 'Transform' has not been declared
  722 |         { Transform::Identity, SPA_META_TRANSFORMATION_None },
      |           ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:723:11: error: 'Transform' has not been declared
  723 |         { Transform::Rot0, SPA_META_TRANSFORMATION_None },
      |           ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:724:11: error: 'Transform' has not been declared
  724 |         { Transform::HFlip, SPA_META_TRANSFORMATION_Flipped },
      |           ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:725:11: error: 'Transform' has not been declared
  725 |         { Transform::VFlip, SPA_META_TRANSFORMATION_Flipped180 },
      |           ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:726:11: error: 'Transform' has not been declared
  726 |         { Transform::HVFlip, SPA_META_TRANSFORMATION_180 },
      |           ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:727:11: error: 'Transform' has not been declared
  727 |         { Transform::Rot180, SPA_META_TRANSFORMATION_180 },
      |           ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:728:11: error: 'Transform' has not been declared
  728 |         { Transform::Transpose, SPA_META_TRANSFORMATION_Flipped90 },
      |           ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:729:11: error: 'Transform' has not been declared
  729 |         { Transform::Rot90, SPA_META_TRANSFORMATION_90 },
      |           ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:730:11: error: 'Transform' has not been declared
  730 |         { Transform::Rot270, SPA_META_TRANSFORMATION_270 },
      |           ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:731:11: error: 'Transform' has not been declared
  731 |         { Transform::Rot180Transpose, SPA_META_TRANSFORMATION_Flipped270 },
      |           ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:734:60: error: 'Transform' was not declared in this scope
  734 | static uint32_t libcamera_transform_to_spa_transform_value(Transform transform)
      |                                                            ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp: In function 'int mmap_init({anonymous}::impl*, {anonymous}::port*, spa_buffer**, uint32_t)':
../spa/plugins/libcamera/libcamera-utils.cpp:791:90: error: 'class libcamera::CameraConfiguration' has no member named 'transform'
  791 |                                 libcamera_transform_to_spa_transform_value(impl->config->transform);
      |                                                                                          ^~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:791:99: error: 'libcamera_transform_to_spa_transform_value' cannot be used as a function
  791 |                                 libcamera_transform_to_spa_transform_value(impl->config->transform);
      |                                                                                                   ^
In file included from ../spa/include/spa/support/log.h:16,
                 from ../spa/plugins/libcamera/libcamera-source.cpp:14:
../spa/plugins/libcamera/libcamera-utils.cpp:793:98: error: 'class libcamera::CameraConfiguration' has no member named 'transform'
  793 |                                 i, b->videotransform->transform, transformToString(impl->config->transform));
      |                                                                                                  ^~~~~~~~~
../spa/include/spa/utils/hook.h:161:50: note: in definition of macro 'spa_callbacks_call'
  161 |                 _f->method((callbacks)->data, ## __VA_ARGS__);                  \
      |                                                  ^~~~~~~~~~~
../spa/include/spa/support/log.h:228:22: note: in expansion of macro 'spa_interface_call'
  228 |                 if (!spa_interface_call(_if,                            \
      |                      ^~~~~~~~~~~~~~~~~~
../spa/include/spa/support/log.h:255:9: note: in expansion of macro 'spa_log_logt'
  255 |         spa_log_logt(l,lev,t,__FILE__,__LINE__,__func__,__VA_ARGS__)
      |         ^~~~~~~~~~~~
../spa/include/spa/support/log.h:258:9: note: in expansion of macro 'spa_logt_lev'
  258 |         spa_logt_lev(l,lev,SPA_LOG_TOPIC_DEFAULT,__VA_ARGS__)
      |         ^~~~~~~~~~~~
../spa/include/spa/support/log.h:269:33: note: in expansion of macro 'spa_log_lev'
  269 | #define spa_log_debug(l,...)    spa_log_lev(l,SPA_LOG_LEVEL_DEBUG,__VA_ARGS__)
      |                                 ^~~~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:792:25: note: in expansion of macro 'spa_log_debug'
  792 |                         spa_log_debug(impl->log, "Setting videotransform for buffer %d to %u (from %s)",
      |                         ^~~~~~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:793:66: error: 'transformToString' was not declared in this scope
  793 |                                 i, b->videotransform->transform, transformToString(impl->config->transform));
      |                                                                  ^~~~~~~~~~~~~~~~~
../spa/include/spa/utils/hook.h:161:50: note: in definition of macro 'spa_callbacks_call'
  161 |                 _f->method((callbacks)->data, ## __VA_ARGS__);                  \
      |                                                  ^~~~~~~~~~~
../spa/include/spa/support/log.h:228:22: note: in expansion of macro 'spa_interface_call'
  228 |                 if (!spa_interface_call(_if,                            \
      |                      ^~~~~~~~~~~~~~~~~~
../spa/include/spa/support/log.h:255:9: note: in expansion of macro 'spa_log_logt'
  255 |         spa_log_logt(l,lev,t,__FILE__,__LINE__,__func__,__VA_ARGS__)
      |         ^~~~~~~~~~~~
../spa/include/spa/support/log.h:258:9: note: in expansion of macro 'spa_logt_lev'
  258 |         spa_logt_lev(l,lev,SPA_LOG_TOPIC_DEFAULT,__VA_ARGS__)
      |         ^~~~~~~~~~~~
../spa/include/spa/support/log.h:269:33: note: in expansion of macro 'spa_log_lev'
  269 | #define spa_log_debug(l,...)    spa_log_lev(l,SPA_LOG_LEVEL_DEBUG,__VA_ARGS__)
      |                                 ^~~~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:792:25: note: in expansion of macro 'spa_log_debug'
  792 |                         spa_log_debug(impl->log, "Setting videotransform for buffer %d to %u (from %s)",
      |                         ^~~~~~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:793:98: error: 'class libcamera::CameraConfiguration' has no member named 'transform'
  793 |                                 i, b->videotransform->transform, transformToString(impl->config->transform));
      |                                                                                                  ^~~~~~~~~
../spa/include/spa/utils/hook.h:161:50: note: in definition of macro 'spa_callbacks_call'
  161 |                 _f->method((callbacks)->data, ## __VA_ARGS__);                  \
      |                                                  ^~~~~~~~~~~
../spa/include/spa/support/log.h:232:21: note: in expansion of macro 'spa_interface_call'
  232 |                     spa_interface_call(_if,                             \
      |                     ^~~~~~~~~~~~~~~~~~
../spa/include/spa/support/log.h:255:9: note: in expansion of macro 'spa_log_logt'
  255 |         spa_log_logt(l,lev,t,__FILE__,__LINE__,__func__,__VA_ARGS__)
      |         ^~~~~~~~~~~~
../spa/include/spa/support/log.h:258:9: note: in expansion of macro 'spa_logt_lev'
  258 |         spa_logt_lev(l,lev,SPA_LOG_TOPIC_DEFAULT,__VA_ARGS__)
      |         ^~~~~~~~~~~~
../spa/include/spa/support/log.h:269:33: note: in expansion of macro 'spa_log_lev'
  269 | #define spa_log_debug(l,...)    spa_log_lev(l,SPA_LOG_LEVEL_DEBUG,__VA_ARGS__)
      |                                 ^~~~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:792:25: note: in expansion of macro 'spa_log_debug'
  792 |                         spa_log_debug(impl->log, "Setting videotransform for buffer %d to %u (from %s)",
      |                         ^~~~~~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:793:66: error: 'transformToString' was not declared in this scope
  793 |                                 i, b->videotransform->transform, transformToString(impl->config->transform));
      |                                                                  ^~~~~~~~~~~~~~~~~
../spa/include/spa/utils/hook.h:161:50: note: in definition of macro 'spa_callbacks_call'
  161 |                 _f->method((callbacks)->data, ## __VA_ARGS__);                  \
      |                                                  ^~~~~~~~~~~
../spa/include/spa/support/log.h:232:21: note: in expansion of macro 'spa_interface_call'
  232 |                     spa_interface_call(_if,                             \
      |                     ^~~~~~~~~~~~~~~~~~
../spa/include/spa/support/log.h:255:9: note: in expansion of macro 'spa_log_logt'
  255 |         spa_log_logt(l,lev,t,__FILE__,__LINE__,__func__,__VA_ARGS__)
      |         ^~~~~~~~~~~~
../spa/include/spa/support/log.h:258:9: note: in expansion of macro 'spa_logt_lev'
  258 |         spa_logt_lev(l,lev,SPA_LOG_TOPIC_DEFAULT,__VA_ARGS__)
      |         ^~~~~~~~~~~~
../spa/include/spa/support/log.h:269:33: note: in expansion of macro 'spa_log_lev'
  269 | #define spa_log_debug(l,...)    spa_log_lev(l,SPA_LOG_LEVEL_DEBUG,__VA_ARGS__)
      |                                 ^~~~~~~~~~~
../spa/plugins/libcamera/libcamera-utils.cpp:792:25: note: in expansion of macro 'spa_log_debug'
  792 |                         spa_log_debug(impl->log, "Setting videotransform for buffer %d to %u (from %s)",
      |                         ^~~~~~~~~~~~~
```

--- a/pkgs/development/libraries/pipewire/default.nix
+++ b/pkgs/development/libraries/pipewire/default.nix
@@ -105,6 +105,14 @@ let
       ./0060-libjack-path.patch
       # Move installed tests into their own output.
       ./0070-installed-tests-path.patch
+
+      # Remove for release post 1.0.1:
+      (fetchpatch {
+        # https://gitlab.freedesktop.org/pipewire/pipewire/-/merge_requests/1750
+        name = "pipewire-spa-libcamera-use-cameraconfiguration-orientation-pr1750.patch";
+        url = "https://gitlab.freedesktop.org/pipewire/pipewire/-/merge_requests/1750.patch ";
+        hash = "sha256-Ugg913KZDKELnYLwpDEgYh92YPxccw61l6kAJulBbIA=";
+      })
     ];
 
     strictDeps = true;

